projects/mysadna2/src/app/data.service.ts

```diff
@@ -0,0 +1,74 @@
+import { Injectable } from '@angular/core';
+import { Observable, of } from 'rxjs';
+import { HttpClient, HttpHeaders } from '@angular/common/http';
+import { catchError, map, tap } from 'rxjs/operators';
+
+import { User } from './user';
+import { Project } from './project';
+
+import {USERS} from './mock-users';
+import {PROJECTS} from './mock-projects';
+
+const httpOptions = {
+  headers: new HttpHeaders({ 'Content-Type': 'application.json' })
+};
+
+@Injectable({
+  providedIn: 'root'
+})
+export class DataService {
+  //private usersUrl = 'api/users'; //For web api
```

Comment:
Right now I'm using local consts for data, but later on I'll need to fetch it from a server or firebase, so I decided to leave those routes in case I'd need them.

Reply:
Sometimes it's ok, but usually it'd better to not leave commented code in a release.
It may confuse reviewers. It makes code dirty and difficult to read. And you may forget about the comment and the commented code will there forever.

If you need to leave message for future development, you can write "TODO" comments.
As I can see you're already using it. Good job!